### ISSUE TITLE
Counter-Strike: Smallest crosshair still to large for modern monitors and resolutions.

### ISSUE THREAD

**Issue by DerRidda** [ValveSoftware/halflife#1925]
This is a feature request, not a bug and certainly not Linux specific but still worthwhile:

The current smallest possible setting for the crosshair in CS is still extremely large on modern big screen displays and high resolutions, it's not unusable but can be hard to focus on for some (like me). It seems that this is a problem widely recognized in the CS 1.6 community and people resort to a crude workaround to get a smaller corsshair that needs to be applied every time one starts the game. ( http://youtu.be/SkXTOxAjQUo) 

If it isn't too much to ask I would love to see one or more of the following features.

1) Introduce a few smaller presets below the current "small" one in the crosshair options

2) Implement an optional crosshair dot in the center of the crosshair much like CS:GO ( This would actually guarantee that there is always a central point of focus no matter how big the screen and/or resolution is.)

3) Give us a configurable scaling factor much like both CS:S and CS:GO have (cl_crosshairscale I think)

I know this is a bit outside the scope of "just bringing CS to Linux and making it work" but if it can be done easily it would really be nice to have.

**Comment by bernky** [comment]
Bump

**Comment by Lordomus** [comment]
Hey valve. We still kinda need this. We enjoy classics but the res are growing, and the crosshair too. You cant play on 700 pixel corsshair.

**Comment by CS-PRO1** [comment]
Yep this issue is painful for sure, a center dot option would be great as well. Good luck :)

**Comment by MarioAR1** [comment]
Bump ^^

**Comment by N33Gr00** [comment]
Hopefully it comes out in the next update.

**Comment by alexplesoiu** [comment]
bump

**Comment by FourLion4L** [comment]
Bump.

**Comment by RadicalPolio** [comment]
BUMP, I STILL WANT THIS!
